src/gui/design.py

```diff
@@ -13,6 +11,9 @@
 MAX_DESCRIPTION_LENGTH = 256
 TIMEOUT_TIMER_PERIOD_MS = 60 * 1000
 TEMPORARY_STORAGE_LABEL_DEFAULT_TEXT = 'Describe what you want to store here...'
+MEMBER_NUMBER_LENGTH = 4
+#PIN_CODE_LENGTH = 6
```

Comment:
It should be `MEMBER_NUMBER_MINIMUM_LENGTH`. In about 10 years time we will have more than 10000 unique members, and thus more than 4 numbers for the member number :)